code/game/turfs/walls/wall_types.dm

```diff
@@ -273,11 +273,24 @@
 //	icon_state = "title_holiday"
 	layer = FLY_LAYER
 	pixel_x = -64
+	//random title as starting point so it isnt choking every time you see it.
+	var/list/total_titles
+	///Gets the current title number. It sets itself with the proc below, no need to touch.
+	var/icon/current_title
+	///The delay between title changes.
+	var/switch_time = 1 MINUTES
```

Comment:
Why does this need to be a variable?

Reply:
so it can be adjusted easier if wanted later.

Reply:
Why does it need to be a var? Only make things vars if you intend to change it on a subtype or such